### PR TITLE
Fix processing of labels without description

### PR DESCRIPTION
**Pull request by jessebrennan**

### CODE REVIEW

File: sync_labels.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import argparse
 from github import (
     Github,
+    GithubException,
```
Comment (amarjandu) [inline sync_labels.py]: Should this be part of the first commit commit that reduces redundancy?

**Review by amarjandu — CHANGES_REQUESTED** [review]
I think there is an issue with the import formatting that was done in the first commit. Other than that it looks great.

File: sync_labels.py (hunk 1 above)
Comment (jessebrennan) [reply sync_labels.py]: Nice catch. Actually, it should be totally removed. Catching just the `GithubException` was too limiting. Other exceptions were being raised that halted the whole process when it should have just carried on.

**Review by amarjandu — APPROVED** [review]

**Comment by jessebrennan** [comment]
@theathorn I'm wondering if it is even worth @hannes-ucsc's time to have him review this.

**Comment by theathorn** [comment]
That's @hannes-ucsc call..

**Review by hannes-ucsc — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
@jessebrennan, OK to merge.

**Pull request merged**